content/applications/websites/website/pages/menus.rst

```diff
@@ -0,0 +1,88 @@
+- **create a regular drop-down menu** by dragging and dropping the sub-menu items to the right,
+  underneath their parent menu.
+
+.. image:: menus/menu-editor.png
+   :alt: Menu editor with sub-menus
+
+.. note::
+
+  You can also access the menu editor by clicking :guilabel:`Edit`, selecting any menu item and
+  clicking the :guilabel:`Edit menu` icon.
+
+  .. image:: menus/edit-menu-icon.png
+     :alt: Access the Menu editor while in Edit mode.
+
+Adding regular menu items
+-------------------------
+
+By default, pages are added to the menu as regular menu items when :doc:`they are created
+<../pages>`. You can also add regular menu items from the menu editor by clicking :guilabel:`Add
+Menu Item`. Enter the :guilabel:`Name` and URL of the related page in the pop-up window that appears
+on the screen and click :guilabel:`OK`.
+
+.. tip::
+   In the :guilabel:`URL or Email` field, you can type `/` to search for a page on your website or
+   `#` to search for an existing custom anchor.
+
+Adding mega menus
+-----------------
+
+Mega menus are similar to drop-down menus, but instead of a simple list of sub-menus, they display a
+panel divided into groups of navigation options. This makes them suitable for websites with large
+amounts of content, as they can help include all of your web pages in the menu while still making
+all menu options visible at once. Mega menus can also be structured more visually than regular
+drop-down menu items, for example, through layout, typography, and icons.
+
+.. image:: menus/mega-menu.png
+   :alt: Mega menu in the navigation bar.
+
+To create a mega menu, go to :menuselection:`Website --> Site --> Menu Editor` and click
+:guilabel:`Add Mega Menu Item`. Enter the :guilabel:`Name` of the mega menu in the pop-up, click
+:guilabel:`OK`, then :guilabel:`Save`.
+
+To adapt the options and layout of the mega menu, click it in the navigation bar, then click
+:guilabel:`Edit`. Mega menus are composed of building blocks, which means you can customize each
+component individually using inline formatting, as well as the options available in the
+:guilabel:`Customize` tab in the website builder. For example, you can:
+
+- edit the text directly in the building block;
+- edit an option's URL by selecting the option, then clicking the :guilabel:`Edit link` button in
+  the small preview pop-up. Type `/` to search for a page on your website, or `#` to search for an
+  existing custom anchor.
+
+  .. image:: menus/mega-menu-option.png
+     :alt: Edit a mega menu option.
+
+- move an option by dragging and dropping the related block to the desired position in the mega
+  menu;
+- delete an option by deleting the related block;
+- adapt the general layout of the mega menu by selecting the desired :guilabel:`Template` and
+  :guilabel:`Size` in the :guilabel:`Mega menu` section in the :guilabel:`Customize` tab.
+
+Header and navigation bar appearance
```

Comment:
_NB: the comment is made assuming you leave the structure as it is:_

I would make this a `.. tip::` since it's not solely related to the menu + there isn't enough content to justify an H2 heading, and I don't think it should be on the same level as the menu editor.
If you still would like to keep the heading, you could add a screenshot to make the section longer, maybe showing some of the customization options available.